kie-archetypes/kie-service-spring-boot-archetype/README.md

```diff
@@ -0,0 +1,84 @@
+# KIE Service Spring Boot Archetype
+
+Archetype used to build new KIE Service Spring Boot Applications.
+
+Generate Spring Boot Service app
+-----------------------------------
+1. Build the kie-service-spring-boot-archetype module (mvn clean install)
+2. Change to directory of your choice where you want to build the 
+app from this archetype.
+3. Create your new app from the archetype with:
+```
+mvn archetype:generate 
+   -DarchetypeGroupId=org.kie 
+   -DarchetypeArtifactId=kie-service-spring-boot-archetype 
+   -DarchetypeVersion=7.10.0-SNAPSHOT
+```
+or use this one-liner
+
+```
+mvn archetype:generate -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion=7.10.0-SNAPSHOT
+```
+4. Change the prompted values during the generation as needed (or leave the defaults)
+5. Compile and test your generated base kjar project with 
+```
+mvn clean install
+```
+6. Start your app with
+```
+mvn clean spring-boot:run
+```
+6. Access your app in browser
+```
+http://localhost:8090
+```
+
+Depending on the app type you select when building your app (more details on this below)
+Your starting page should look like this:
+
+![alt text](sample.png?raw=true "Sample index page of your generated app")
+
+Default Basic Authentication
+-----------------------------------
+Your generated app has basic authentication built in. You can always change this later to add/remove users as needed.
+By default you can log in to your application with
+```
+username: user
+password: user
+```
+
+Manage Apps KIE Server capabilities
+-----------------------------------
+You can manage what Kie server capabilities are included in your 
+service app with the property
+```
+-DappType
+```
+If you don't manually specify this property
+it will be set to "default".
+
+Currently there are three possible choices 
+for this property:
+1. default: includes BRM, BPM, Case Management, BPM-UI, and DMN
+2. businessmanagement: includes BRM and DMN
+3. optimizer: includes BRM, BRP, and DMN
```

Comment:
can we change the property name to be:
- bpm 
- brm
- planner
just os it's shorter and easier to grasp